### PR TITLE
changed algorithm in task 7

### PR DESCRIPTION
**Pull request by StanislauBazarnou**

### CODE REVIEW

File: src/main/java/by/epam/learn/main/tasks/Task7.java [hunk 1]
```diff
@@ -5,46 +5,42 @@
...
-
-
-
+        Collections.sort(numbers);
+        printArray(numbers);
 
+        Collections.reverse(numbers);
+        printArray(numbers);
 
+//        for (int j = numbers.size()-1; j > -1; j--) {
```
Comment (EDbarvinsky) [inline src/main/java/by/epam/learn/main/tasks/Task7.java]: Also could be deleted as unused

File: src/main/java/by/epam/learn/main/tasks/Task7.java [hunk 2]
```diff
@@ -5,46 +5,42 @@
...
+//        }
 
 
 
-//        for (int i = 0; i < list.size(); i++) {
-//            list.sort(Comporator.comparing(Integer::length));
-//            System.out.println(list.get(i));
+        //        for (int i = 0; i < numbers.size(); i++) {
+//            numbers.sort(Comporator.comparing(Integer::length));
+//            System.out.println(numbers.get(i));
```
Comment (EDbarvinsky) [inline src/main/java/by/epam/learn/main/tasks/Task7.java]: the same here

File: src/main/java/by/epam/learn/main/tasks/Task7.java (hunk 1 above)
Comment (StanislauBazarnou) [reply src/main/java/by/epam/learn/main/tasks/Task7.java]: Done

Comment (EDbarvinsky) [reply src/main/java/by/epam/learn/main/tasks/Task7.java]: @StanislauBazarnou good job

Comment (EDbarvinsky) [reply src/main/java/by/epam/learn/main/tasks/Task7.java]: @StanislauBazarnou Just for your information - in production development activities ANY fixes should be done in separate PR (pull\merge request)
P.S. feel free to open any other PR for check

Comment (StanislauBazarnou) [reply src/main/java/by/epam/learn/main/tasks/Task7.java]: @EDbarvinsky good point, thanks!

**Pull request merged**